Remove version path from reportsUrl

otech47 opened the pull request:
This commit https://github.com/7eggs/node-toggl-api/commit/d396c1cda5d5f54fc132a24e79c94f1dc76aff33 broke the reports endpoints because the `/api/v2` is added with every `reportsRequest` call. This results in invalid calls to the reports API (404 not found)

![Screen Shot 2022-05-23 at 12 35 32 PM](https://user-images.githubusercontent.com/4914611/169801262-cb622715-03f3-4081-b740-3faba5be8f68.png)

rzfzr commented:
@otech47 sorry you had this issue, I might have not tested this.
I can fix it later today, can you tell me if "api.track." should be kept?

rzfzr commented:
I just noticed that you closed the issue, can you share the reasons?

otech47 commented:
It was a bit strange...

I saw I had the older version with the www.toggl.com endpoint and went to update the package

I ran `npm update toggle-api` and saw my package.json update from `^1.0.1` to `^1.0.2` so I assumed all was good

But then I went to run it and I noticed the repeated `/api/v2/api/v2`

I forked and changed the `reportsUrl` as shown in this PR

Then I changed my package.json to use my fork that had the new commit

```
"toggl-api": "https://github.com/setlife-network/node-toggl-api.git",
```

Then I ran `npm i` and went to run it all again but then the URL left out the `/api/v2` entirely so the endpoint was still invalid

Then I reverted the commit on my fork and reran the installation and then it all worked...

Not sure exactly what happened but I am pointing to my fork that is exactly the same as the master branch of this repo and it works fine

Maybe I missed a step when updating the package version originally but hope that info is helpful
